bangc-ops/mlu_op.h

```diff
@@ -10849,6 +10849,237 @@ mluOpPointsInBoxes(mluOpHandle_t handle,
+ * @param[in] grads_desc
+ * Input. The descriptor of the gradient tensor in the backpropagation process. For detailed
+ * information, see ::mluOpTensorDescriptor_t.
+ * @param[in] grads
+ * Input. Pointer to the MLU memory that stores the gradient tensor.
+ * @param[in] boxes_desc
+ * Input. The descriptor of the bounding boxes tensor. For detailed information, see
+ * ::mluOpTensorDescriptor_t.
+ * @param[in] boxes
+ * Input. Pointer to the MLU memory that stores the bounding boxes tensor.
+ * @param[in] argmax_x_desc
+ * Input. The descriptor of the \b argmax_x tensor that stores the coordinate of x axis. For detailed
+ * information, see ::mluOpTensorDescriptor_t.
+ * @param[in] argmax_x
+ * Input. Pointer to the MLU memory that stores the \b argmax_x tensor. \b argmax_x represents
+ * \b output coordinate of x axis returned by ::mluOpRoiAlign_v2 when \b pool_mode is maximum
+ * pooling mode. When \b pool_mode is average pooling mode, \b argmax_x is NULL.
+ * @param[in] argmax_y_desc
+ * Input. The descriptor of the \b argmax_y tensor that stores the coordinate of y axis. For detailed
+ * information, see ::mluOpTensorDescriptor_t.
+ * @param[in] argmax_y
+ * Input. Pointer to the MLU memory that stores the \b argmax_y tensor. \b argmax_y represents
+ * \b output coordinate of y axis returned by ::mluOpRoiAlign_v2 when \b pool_mode is maximum
+ * pooling mode. When \b pool_mode is average pooling mode, \b argmax_y is NULL.
+ * @param[in] spatial_scale
+ * Input. A scaling factor that specifies how to map the box coordinates in the original image to
+ * the coordinates in the output.
+ * @param[in] sampling_ratio
+ * Input. The number of sampling points in the grid used to compute the output.
+ * @param[in] aligned
+ * Input. A boolean value which determines whether to shift the boxes by 0.5 pixel. If the value
+ * of \b aligned is set to true, the boxes are shifted by 0.5. If the value of \b aligned is set
+ * to false, the boxes are not shifted.
+ * @param[in] pool_mode
+ * Input. The pooling mode which determines to use maximum pooling mode or average
+ * pooling mode. If the value of \b pool_mode is set to 1, the average pooling mode is used. If
+ * the value of \b pool_mode is set to 0, the maximum pooling mode is used.
+ * @param[in] grads_image_desc
+ * Input. The descriptor of the gradients tensor of the original images. For detailed information,
+ * see ::mluOpTensorDescriptor_t.
+ * @param[out] grads_image
+ * Output. Pointer to the MLU memory that stores the \b grads_image tensor .
+ *
+ * @par Return
+ * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM, ::MLUOP_STATUS_ARCH_MISMATCH.
+ *
+ * @par Data Type
+ * - This function supports the following data types for gradient tensor \b grads, boxes tensor
+ *   \b boxes, argmax_x tensor \b argmax_x, argmax_y tensor \b argmax_y and output tensor \b
+ *   grads_image. Data type of all tensors should be the same.
```

Comment:
- The data types of all tensors should be the same.
- The supported data types of input and output tensors are as follows: